ui/src/app/job-details/common/attempt/attempt.component.html

```diff
@@ -0,0 +1,64 @@
+<mat-expansion-panel class="list-header" disabled="disabled">
+  <mat-expansion-panel-header *ngIf="attempt">
+    <mat-panel-title>
+      <div class="task-name">
+        attempt {{ attempt.attemptNumber }}
+      </div>
+    </mat-panel-title>
+    <mat-panel-description>
+      <div class="task-status">
+        <clr-tooltip>
+          <clr-icon clrTooltipTrigger [attr.shape]="getStatusIcon(attempt.executionStatus)" size="18"></clr-icon>
+          <clr-tooltip-content clrPosition="left" clrSize="xs" *clrIfOpen>
+            <span>{{ attempt.executionStatus }}</span>
+          </clr-tooltip-content>
+        </clr-tooltip>
+      </div>
+      <div class="task-start"><jm-datetime [datetime]="attempt.start"></jm-datetime></div>
+      <div class="task-duration">
+        {{ attempt.start | jmDuration: attempt.end }}
+        <clr-tooltip *ngIf="attempt.callCached">
+          <clr-icon clrTooltipTrigger shape="history" size="14" style="color:rgba(0, 0, 0, 0.54)"></clr-icon>
+          <clr-tooltip-content clrPosition="left" clrSize="xs" *clrIfOpen>
+            <span>This attempt was cached</span>
+          </clr-tooltip-content>
+        </clr-tooltip>
+      </div>
+      <div class="task-inputs">
+        <button mat-icon-button *ngIf="hasInputs(attempt)" [mat-menu-trigger-for]="inputMenu" (click)="$event.stopPropagation()">
```

Comment:
This section looks like it might repeat multiple times? Would it be worth abstracting to a component?